src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -791,7 +791,14 @@ static bool ValidateConcatAxis(const InferenceEngine::CNNLayerPtr layer, std::st
                         0,
                         [](InferenceEngine::CNNLayerPtr ptr) {
                             return LayerInfo(ptr).isNonFunctional() || LayerInfo(ptr).isFakeQuantize() ||
-                                   LayerInfo(ptr).isSplit();
+                                   LayerInfo(ptr).isSplit() ||
+                                   (LayerInfo(ptr).isPermute() &&
```

Comment:
Shouldn't we verify the original layout as well? And what about 3D cases? What if the skipped permutation is an "artificial" one around a convolution?
I also suggest to create a function(s) from this piece of code, which is similar to one used a few lines above.

Reply:
The solution has been changed, so this comment is already obsolete.